README.md

```diff
@@ -258,6 +258,63 @@ Airbrake.configure do |c|
 end
 ```
 
+#### blacklist_keys
+
+Specifies which keys in the payload (parameters, session data, environment data,
+etc) should be filtered. Before sending an error, filtered keys will be
+substituted with the `[Filtered]` label.
+
+It accepts Strings, Symbols & Regexps, which represent keys to be filtered.
```

Comment:
Changed to `It accepts Strings, Symbols & Regexps, which represent keys of values to be
filtered.`, thanks!